src/main/java/org/traccar/database/OpenIdProvider.java

```diff
@@ -111,13 +114,14 @@ public OpenIdProvider(Config config, LoginService loginService, HttpClient httpC
 
         adminGroup = config.getString(Keys.OPENID_ADMIN_GROUP);
         allowGroup = config.getString(Keys.OPENID_ALLOW_GROUP);
+        groupsClaimName = config.getString(Keys.OPENID_GROUPS_CLAIM_NAME);
     }
 
     public URI createAuthUri() {
         Scope scope = new Scope("openid", "profile", "email");
 
         if (adminGroup != null) {
-            scope.add("groups");
+            scope.add(Objects.requireNonNullElse(groupsClaimName, "groups"));
```

Comment:
Can we just use `"groups"` as the default value?

Reply:
Yes. Absolutely. requireNonNullElse sets default value if groupsClaimName is not provided.

Reply:
Yes, but we have a standard mechanism we use everywhere else.

Reply:
if condition?

Reply:
No, configuration keys can have default values.